internal/store/postgres/organization_repository.go

```diff
@@ -189,6 +190,10 @@ func (r OrganizationRepository) Create(ctx context.Context, org organization.Org
 		return organization.Organization{}, fmt.Errorf("%w: %s", parseErr, err)
 	}
 
+	if err = validateMetadataSchema(marshaledMetadata, orgMetaSchemaName); err != nil {
```

Comment:
Validation should happen in grpc handlers, that is `v1beta1` I think. By the time the request has reached postgres, it might have already gone through lot of services. We need to fail the call early if metadata is bad.